src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.Interface.cs

```diff
@@ -87,6 +87,26 @@ partial class CreateNewImage
     /// </summary>
     public ITaskItem[] EntrypointArgs { get; set; }
 
+    /// <summary>
+    /// Default arguments passed. These can be overridden by the user when the container is created.
+    /// </summary>
+    public ITaskItem[] DefaultArgs { get; set; }
```

Comment:
This roughly corresponds to what https://github.com/dotnet/sdk-container-builds/issues/11 called `ContainerCommand`.